src/panels/lovelace/editor/config-elements/hui-statistics-graph-card-editor.ts

```diff
@@ -154,6 +221,9 @@ export class HuiStatisticsGraphCardEditor
           .hass=${this.hass}
           .pickStatisticLabel=${`Add a statistic`}
           .pickedStatisticLabel=${`Statistic`}
+          .includeDisplayUnitOfMeasurement=${
+            displayUnit !== undefined ? [displayUnit] : undefined
```

Comment:
This is maybe a bit meh, should we allow passing non-arrays? Or have two properties, `includeDisplayUnitOfMeasurement` for a single string and `includeDisplayUnitOfMeasurements` for an array of strings?

Reply:
By creating an array in render, it will cause an update down the line every update because `[] !== []`